### ISSUE TITLE
Bot endpoint to get current page

### ISSUE THREAD

**Issue by kylepolich** [data-skeptic/dataskeptic.com#186]
I would like for the bot (inside the iframe) to make an API request to determine if the user is coming from the site (they can also find the Bot on Skype).  If the user is coming from the site, can we tell the exact URL of the page they are on?

Is it possible?

**Comment by glozhkovoy** [comment]
Yes, we can do that. Bot can post message to our app requesting current page, and we can return ```location.href``` so bot can check url.
